publisher-sdk/src/main/java/com/criteo/publisher/privacy/UserPrivacyUtil.java

```diff
@@ -148,4 +152,42 @@ public void storeMopubConsent(@Nullable String mopubConsent) {
   public String getMopubConsent() {
     return safeSharedPreferences.getString(MOPUB_CONSENT_SHARED_PREFS_KEY, "");
   }
+
+  /**
+   * Indicate if user didn't give its consent for CSM when applicable.
+   *
+   * <ul>
+   *   <li>
+   *     IABTCF_PublisherRestrictions{ID}: publisher restrictions on purpose {ID} (only for ID=1)
```

Comment:
I feel we should explain somewhere what purpose 1 means. That way, the reader/maintainer understands why we only care about `ID=1`. A link to the list of Purpose IDs would be great.